wpilib/wpilib/iterativerobotbase.py

```diff
@@ -148,34 +155,52 @@ def testPeriodic(self):
     def loopFunc(self):
         """Call the appropriate function depending upon the current robot mode"""
 
```

Comment:
nit: There shouldn't be a blank line between a method docstring and the method body.